src/doc/guide.md

```diff
@@ -92,6 +92,8 @@ class="s1">     Fresh</span> hello_world v0.1.0 (file:///path/to/project/hello_w
 class="s1">   Running</span> `target/debug/hello_world`
 Hello, world!</code></pre>
 
+To pass some arguments to your program, you can use `cargo run first_argument second_argument`. If flags are being passed to the command being invoked, please to use `cargo run -- --a_flag -b an_argument`.
```

Comment:
The second sentence here may want to be worded along the lines of:

> If flags are being passed to the executable being run a `--` separator can be used to tell Cargo which flags go where, for example: `cargo run -- --foo -b bar`.

Also, can you be sure to wrap this line to 80 chars?